bin/run_tests

```diff
@@ -18,15 +18,13 @@
 # MODULE_LIST determines the order in which modules are run.
 # Quicker tests should run before slower ones.
 MODULE_LIST="
-    signing
     cli
     rest_api
     settings
     identity
     block_info
     battleship
     validator
-    python_sdk
     rust_sdk
```

Comment:
The rust_sdk is left behind. I didn't understand the reason.

Reply:
This will get handled in a different PR.